src/ConnectedMode/Binding/IBindingConfigProvider.cs

```diff
@@ -47,14 +47,10 @@ public interface IBindingConfigProvider
     /// For C++ it will be in a json file in a Sonar-specific format</remarks>
     public interface IBindingConfig
     {
-        /// <summary>
-        /// Returns a list of all of the solution-level files that should exist if the solution is correctly bound. The files might not exist on disk. List cannot be null or empty.
-        /// </summary>
-        IEnumerable<string> SolutionLevelFilePaths { get; }
```

Comment:
The property existed so the `SolutionBindingOperation` could work out which directories to create -> no longer required.